docs/egress-firewall.md

```diff
@@ -0,0 +1,41 @@
+# EgressFirewall
+
+## Introduction
+
+The EgressFirewall feature enables a cluster administrator to
+limit the external hosts that a pod in a project can access.
+The EgressFirewall object rules apply to all pods that share
+the namespace with the egressfirewall object. A namespace only
+supports having one EgressFirewallObject.
+
+## Example
+
+The yaml  below is an example of a simple egressFirewall object
+
+```yaml
```

Comment:
In this example yaml, I don't see the protocol/port. As per the CRD, the protocol is a required field of the rule?

Reply:
@abhat the port is an optional field  the protocol is required if a port is specified and the port number is optional

Reply:
nit: Gotcha, having said that, when we are providing an example CR, we should try to capture all the fields including the optional ones.

Reply:
done